steps/27/README.md

```diff
@@ -97,7 +97,11 @@ import "./model/formatter";
 
 ### webapp/test/Test.qunit.html \(New\)
 
-We also need a generic test page to run our tests. This is necessary as the namespace of the test code needs to be registered via the `data-sap-ui-resource-roots` attribute. The page will also be referenced in the test suite, which we're going to create next.
+We also need a generic test page that will be used to run individual tests.
+
```

Comment:
Can we have here one additional sentence about the src="../resources/sap/ui/test/starter/runTest.js" script that is initialized in the script tag? Would be good to also mention what this script is for.

Reply:
Done